prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Events</title>
+</head>
+<body>
+  <button class="hello-world-btn">Hello World!</button>
+  <script>
+    // Your code here
+    document.addEventListener('DOMContentLoaded', content);
+    function content() {
+
+    }
+    console.log('content-loaded!');
+
+    window.addEventListener('mousemove', mouse);
+    function mouse() {
+
+    }
+    console.log('mouse is moving!');
+
```

Comment:
Hi Jeff, try moving your console logs into your functions so that they execute when the actual events happen